netbox/core/api/views.py

```diff
@@ -71,3 +82,251 @@ class ObjectChangeViewSet(ReadOnlyModelViewSet):
     queryset = ObjectChange.objects.valid_models()
     serializer_class = serializers.ObjectChangeSerializer
     filterset_class = filtersets.ObjectChangeFilterSet
+
+
+class QueueListView(APIView):
```

Comment:
These list views should replicate the existing API convention, which enables pagination of the results:

```json

{
    "count": 24,
    "next": null,
    "previous": null,
    "results": [...]
}
```